### ISSUE TITLE
Audio Muting for the Daily Wire

### ISSUE THREAD

**Issue by richardfrost** [richardfrost/AdvancedProfanityFilter#429]
#### :movie_camera: The Site
www.dailywire.com

#### :pencil: Additional notes
Requested by a user on the Chrome Web Store

**Comment by richardfrost** [comment]
This config seems to work. I'll get it added to the next version:

```json
{
  "www.dailywire.com": [
    {
      "mode": "cue"
    }
  ]
}
```

**Comment by richardfrost** [comment]
This is included in [Version 2.37.0](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.37.0). If anyone has a chance to test this out, feel free to reply here to report back how it works for you. Thanks!
